bus/api.go

```diff
@@ -85,6 +85,12 @@ type ObjectsResponse struct {
 	Object  *object.Object `json:"object,omitempty"`
 }
 
+// AddObjectRequest is the request type for the /object/*key PUT endpoint.
+type AddObjectRequest struct {
+	Object        object.Object `json:"object"`
+	UsedContracts map[consensus.PublicKey]types.FileContractID
```

Comment:
missing `json` tag

Reply:
maybe add a custom type for the map? it's quite lengthy and messes up readability in methods/interfaces

Reply:
Tbh I'm not a huge fan of custom types for data structures unless they are nested or really horrible to look at. This seems still within reason to me but if everyone feels the same I can change it of course.

Reply:
nah leave it, just a `nit`